brainstormforce/ultimate-addons-for-gutenberg
# Fix : Call To Action - Anchor does not save

### Description

Trello - https://trello.com/c/uH3oGxl7/90-call-to-action-anchor-does-not-save

### Types of changes




Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Created CTA block in master.
- Checkout to this branch and added anchor to the CTA block.
- Check front-end is id is visible or not.

### Checklist:
